haystack/nodes/prompt/invocation_context_mapper.py

```diff
@@ -0,0 +1,352 @@
+from typing import Optional, List, Dict, Any, Tuple, Union, Callable
+
+import logging
+
+from haystack.nodes.base import BaseComponent
+from haystack.schema import Document, MultiLabel
+
+
+logger = logging.getLogger(__name__)
+
+
+def rename(value: Any) -> Tuple[Any]:
+    """
+    Identity function. Can be used to rename values in the invocation context without changing them.
+
+    Example:
+
+    ```python
+    assert rename(1) == (1, )
+    ```
+    """
+    return (value,)
+
+
+def expand_value_to_list(value: Any, target_list: List[Any]) -> Tuple[List[Any]]:
```

Comment:
Shouldn't this one be "value_to_list" to be consistent with other names (like "documents_to_strings")?

Reply:
Right, good idea!